.github/workflows/ci-ipfs.yml

```diff
@@ -0,0 +1,82 @@
+name: Deploy IPFS Base
+
+on:
+  workflow_call:
+    inputs:
+      ipfs_folder:
+        description: 'IPFS build folder'
+        default: 'out/'
+        type: string
+      W3S_TOKEN:
+        description: 'web3storage token'
+        type: string
+      GW3_TOKEN:
+        description: 'gateway3 token'
+        type: string
+
+permissions:
+  contents: write
+
+jobs:
+  deploy-ipfs:
+    runs-on: ubuntu-latest
+    name: Build and deploy
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Install node
+        uses: actions/setup-node@v3
```

Comment:
i'd suggest changing this workflow just to `action` to make it possible not depend on package manager or any other ways of installing dependencies

- create a new workflow in every repo
- install dependencies
- run ipfs action that do another work

Reply:
Do you propose to remove the installation version of the node from this workflow?

Reply:
node and dependencies too